README.md

```diff
@@ -18,8 +18,8 @@ On OS X, the latest release of Libfabric can be installed using the
 [Homebrew](https://github.com/Homebrew/homebrew) package manager using the
 following command:
 
-```
-$ brew install libfabric
+```bash
+brew install libfabric
```

Comment:
The convention is to put '$' before the shell command line. No need to change that unless there is a strong reason to do so. This applies to other similar changes in this commit.